src/views/Home/components/Menu/sandwich.svg

```diff
@@ -0,0 +1 @@
092,-0.396 2.265,0.238 3.6,1.638 2.137,2.243 7.582,1.196 9.11,0.609 1.547,-0.596 2.903,-0.985 2.498,-2.576 -0.089,-0.35 0.722,-0.654 1.548,-0.364 1.388,0.487 3.395,1.694 3.75,3.018 0.991,0.173 1.734,-0.102 2.305,-0.858 0.284,-0.373 1.254,-0.463 1.669,0.025 0.733,0.861 1.934,3.505 0.233,5.774 C 28.635,1.823 25.682,3.197 19.05,3.752 12.417,4.306 3.883,3.645 0,0"/></g><g transform="translate(31,10.959)" id="g48"><path id="path50" style="fill:#ffcc4d;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 0,0 c 0,-0.529 -0.895,-0.959 -2,-0.959 l -22,0 c -1.104,0 -2,0.43 -2,0.959 l 0,10.551 c 0,0.53 0.896,0.49 2,0.49 l 22,0 c 1.105,0 2,0.04 2,-0.49 L 0,0 Z"/></g><g transform="translate(33.3105,16.0127)" id="g52"><path id="path54" style="fill:#be1931;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 0,0 c 0,-2.768 -4.586,-5.013 -10.242,-5.013 -5.658,0 -10.243,2.245 -10.243,5.013 0,2.769 4.585,5.013 10.243,5.013 C -4.586,5.013 0,2.769 0,0"/></g><g transform="translate(33.874,20.2793)" id="g56"><path id="path58" style="fill:#f4900c;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 0,0 -6.909,4.226 -6.907,4.228 c -1.526,0.934 -4.001,0.934 -5.527,0 0,0 -5.527,-3.382 -6.908,-4.228 -1.382,-0.845 -0.691,-1.268 -2.072,-2.114 -1.382,-0.844 -2.072,-0.422 -3.454,-1.267 -1.383,-0.845 -1.526,-2.447 0,-3.382 l 6.907,-4.228 6.909,-4.227 c 1.525,-0.935 4.145,-0.848 5.524,0 1.383,0.845 0.693,1.266 2.074,2.114 1.381,0.844 2.071,0.422 3.454,1.266 C -5.526,-6.765 0,-3.385 0,-3.385 1.524,-2.45 1.525,-0.935 0,0"/></g><g transform="translate(35,20.4844)" id="g60"><path id="path62" style="fill:#f4900c;fill-opacity:1;fill-rule:nonzero;stroke:none" d="M 0,0 -34,0.516 -34,-1.109 0.018,-1.901 0,0 Z"/></g><g transform="translate(35,21.4844)" id="g64"><path id="path66" style="fill:#f4900c;fill-opacity:1;fill-rule:nonzero;stroke:none" d="M 0,0 -34,0.516 -34,-1.109 0.018,-1.901 0,0 Z"/></g><g transform="translate(33.873,23.2793)" id="g68"><path id="path70" style="fill:#ffac33;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 0,0 -6.908,4.226 -6.907,4.228 c -1.526,0.934 -4.001,0.934 -5.527,0 0,0 -5.527,-3.382 -6.908,-4.228 -1.381,-0.845 -0.691,-1.268 -2.072,-2.114 -1.382,-0.844 -2.072,-0.422 -3.454,-1.267 -1.383,-0.845 -1.526,-2.447 0,-3.382 l 6.907,-4.228 6.909,-4.227 c 1.526,-0.935 4.145,-0.848 5.524,0 1.383,0.845 0.693,1.266 2.074,2.114 1.381,0.844 2.071,0.422 3.454,1.266 C -5.525,-6.765 0,-3.385 0,-3.385 1.525,-2.45 1.526,-0.935 0,0"/></g><g transform="translate(32.8496,22.4316)" id="g72"><path id="path74" style="fill:#ffd983;fill-opacity:1;fill-rule:nonzero;stroke:none" d="m 0,0 -7.072,4.229 -7.071,4.227 c 0,0 -1.506,0.791 -2.828,0 0,0 -5.766,-3.447 -6.963,-4.162 -1.196,-0.716 -1.033,-1.464 -2.23,-2.179 -1.196,-0.716 -2.338,-0.553 -3.535,-1.267 -1.414,-0.847 0,-1.694 0,-1.694 l 7.071,-4.227 7.071,-4.228 c 0,0 1.414,-0.844 2.828,0 1.198,0.717 0.926,1.398 2.122,2.115 1.197,0.716 2.447,0.619 3.644,1.334 1.197,0.713 6.963,4.163 6.963,4.163 0,0 1.321,0.9 0,1.689"/></g></g></g></g></svg>
```

Comment:
It would be better to reference the existing file, rather than making a copy here.